src/quill-cursors/cursor.ts

```diff
@@ -77,9 +84,22 @@ export default class Cursor {
     this._el.parentNode.removeChild(this._el);
   }
 
+  public toggleNearCursor(pointX: number, pointY: number): boolean {
+    const {left, right, top, bottom} = this._getCoordinates();
+
+    const isXNear = pointX - right <= 0 && pointX - left >= 0;
```

Comment:
I think I personally find this more readable (same with `y`)

```suggestion
    const isXNear = pointX >= left && pointX <= right;
```